sql/testdata/builtin_function

```diff
@@ -332,3 +332,56 @@ SELECT extract(nanosecond from timestamp '2001-04-10 12:04:59.34565423')
 
 query error extract: unsupported timespan: nansecond
 SELECT extract(nansecond from timestamp '2001-04-10 12:04:59.34565423')
+
+query T
+SELECT abs(-1.0)
+----
+1
+
+query T
+SELECT abs(-0.0)
+----
+0
+
+query T
+SELECT abs(1.0)
+----
+1
+
+query error abs: abs of MinInt64 is not defined
+SELECT abs(-9223372036854775808)
+
+query T
+SELECT abs(-1)
+----
+1
+
+query T
+SELECT abs(0)
+----
+0
+
+query T
+SELECT abs(1)
+----
+1
+
+query T
+SELECT pi()
+----
+3.141592653589793
+
+query T
+SELECT sin(0.0)
+----
+0
+
+query T
+SELECT sin(1.0)
+----
+0.8414709848078965
+
+query T
+SELECT sin(1::float)
```

Comment:
Replaced with `abs(sin(pi())) < 1e-12 == true`, let me know if I should kill the new version of the test, if it's too dense for example.

Wrote it like this, since the numeric value of `sin(pi())` is very close to zero, and the implementation of `sin` is platform dependent. Therefore I'm expecting slightly different results for `sin(pi())`, depending on the platform.